### PR TITLE
Added missing InputFormatters and OData query values to try in swagger UI

### PR DESCRIPTION
**Pull request by AlbertoMonteiro**

### CODE REVIEW

File: WeatherAPI2/Startup.cs [hunk 1]
```diff
@@ -74,16 +74,13 @@ IEdmModel GetEdmModel()
 
         private static void SetOutputFormatters(IServiceCollection services)
         {
-            services.AddMvcCore(options =>
+            services.AddMvcCore(op =>
```
Comment (hassanhabib) [inline WeatherAPI2/Startup.cs]: Why did you shorten this out?

Comment (AlbertoMonteiro) [reply WeatherAPI2/Startup.cs]: There is no reason, I just copied from my application code, that is written in this way, I will change

Comment (AlbertoMonteiro) [reply WeatherAPI2/Startup.cs]: Fixed

**Comment by AlbertoMonteiro** [comment]
😕

**Pull request closed without merging**